cvat/apps/engine/tests/test_rest_api.py

```diff
@@ -5293,6 +5293,84 @@ def test_api_v1_server_share_no_auth(self):
         self.assertEqual(response.status_code, status.HTTP_401_UNAUTHORIZED)
 
 
+class ServerShareDifferentTypesAPITestCase(APITestCase):
+    def setUp(self):
+        self.client = APIClient()
+
+    @classmethod
+    def setUpTestData(cls):
+        create_db_users(cls)
+
+    @staticmethod
+    def _create_shared_files():
+        main_path = os.path.join(settings.SHARE_ROOT, "data1")
+        os.makedirs(main_path)
+        main_images = ["people.jpeg", "street_1.jpeg", "street_2.jpeg", "street_3.jpeg"]
+        for img in main_images:
+            image = Image.new('RGB', size=(100, 50))
+            image.save(osp.join(main_path, img), "JPEG")
+        image.save(osp.join(main_path, "street.png"), "PNG")
```

Comment:
Can we get file extension from image name (with os.path.splitext) and put the operator to the loop above?